src/app/pages/questions/components/question/question.component.html

```diff
@@ -77,6 +77,8 @@
           [min]="question.range.min"
           [max]="question.range.max"
           [step]="question.range.step"
+          [labelLeft]="question.range.labelLeft"
+          [labelRight]="question.range.labelRight"
```

Comment:
how will this be taken from redcap? does the script need updating too?

Reply:
No these are automatically by redcap to the `select_choices_or_calculations` array similar to the `range` type. And it's already currently being parsed and added to `question.range` if this array exists.

Reply:
ok thanks